packages/card/src/vaadin-card.js

```diff
@@ -279,10 +280,47 @@ class Card extends ElementMixin(ThemableMixin(PolylitMixin(LitElement))) {
     `;
   }
 
+  static get properties() {
+    return {
+      /**
+       * The title of the card. When set, any custom slotted title is removed and this string-based title is used instead. If this title is used, an `aria-labelledby` attribute that points to the generated title element is set.
+       *
+       * @type {String}
+       */
+      cardTitle: {
+        type: String,
+        notify: true,
+        observer: '__cardTitleChanged',
+      },
+
+      /**
+       * Sets the heading level (`aria-level`) for the string-based title. If not set, the level defaults to 2. Setting values outside the range [1, 6] can cause accessibility issues.
+       *
+       * @type {Number}
+       */
```

Comment:
Let's add `@attr {number} title-heading-level` here and in the `.d.ts` file.

Reply:
Done.